config/routes.rb

```diff
@@ -63,8 +63,8 @@
 
   # Funding Form pages
   get "/funding-form" => "funding_form#index"
-  get "/funding-form/organisation-name" => "funding_form/organisation_name#show"
-  post "/funding-form/organisation-name" => "funing_form/organisation_name#submit"
+  get "/funding-form/organisation-type" => "funding_form/organisation_type#show"
+  post "/funding-form/organisation-type" => "funing_form/organisation_type#submit"
```

Comment:
Looks like my error in a previous PR, but this line has `funing_form` instead of `funding_form`.  Probably a good time to fix that.